### PR TITLE
Stop discarding result output from MR jobs

### PR DESCRIPTION
**Pull request by charleso**

### CODE REVIEW

File: ivory-storage/src/main/scala/com/ambiata/ivory/storage/repository/Recreate.scala [hunk 1]
```diff
@@ -149,7 +149,7 @@ object Recreate { outer =>
       partitions    <- ScoobiAction.fromHdfs(Hdfs.globFiles(from.toIvoryLocation(Repository.factset(id)).toHdfsPath, "*/*/*/*/*").filterHidden)
       version        = FactsetVersionTwo
       _             <- {
-        ScoobiAction.safe(RecreateFactsetJob.run(configuration, version, dictionary, namespaces, partitions, to.toIvoryLocation(Repository.factset(id)).toHdfsPath, reducerSize, codec))
+        ScoobiAction.fromRIO(RecreateFactsetJob.run(configuration, version, dictionary, namespaces, partitions, to.toIvoryLocation(Repository.factset(id)).toHdfsPath, reducerSize, codec))
```
Comment (charleso) [inline ivory-storage/src/main/scala/com/ambiata/ivory/storage/repository/Recreate.scala]: Obviously a merge conflict when Russell merges

**Comment by markhibberd** [comment]
:lock:

**Comment by nhibberd** [comment]
@charleso This going in soon or? :walking:

File: ivory-storage/src/main/scala/com/ambiata/ivory/storage/repository/Recreate.scala (hunk 1 above)
Comment (charleso) [inline ivory-storage/src/main/scala/com/ambiata/ivory/storage/repository/Recreate.scala]: > Obviously a merge conflict when Russell merges

@raronson Sorry russel. :)

**Pull request merged**